File: Lab 4/t1.c
#include <stdio.h>

int main (void)
{
  int a = 0, b = 0;
  puts ("Please input two integers:");
  scanf ("%d %d", &a, &b);
  printf ("Between %d and %d, %d is greater.\n", a, b, (a >= b ? a : b));
  return 0;
}

File: Lab 4/t1.go
package main

import "fmt"

func main() {
  var a, b int = 0, 0
  fmt.Println("Please input two integers:")
  fmt.Scanf("%d %d", &a, &b)
  fmt.Printf("Between %d and %d, %d is greater.\n", a, b, bigger (a, b))
}

func bigger(a, b int) int {
  if a >= b {
    return a
  } else {
    return b
  }
}
